blocks/feed-newsroom/feed-newsroom.js

```diff
@@ -30,6 +30,9 @@ const resultParsers = {
           } else if (fieldName === 'title') {
             div.classList.add('title');
             div.textContent = result[fieldName];
+          } else if ((fieldName === 'description') && (locale === 'jp')) {
```

Comment:
Probably we should do this for all locales. For e.g. an english article can have same problem with long descriptions.